src/love.cpp

```diff
@@ -292,4 +292,11 @@ int main(int argc, char **argv)
 	return retval;
 }
 
+#ifdef __MINGW32__
+int WinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, LPSTR lpCmdLine, int nShowCmd)
```

Comment:
Isn't the SDL2main library supposed to handle this?

Reply:
I didn't know this, fixed in df108cb.